src/components/projects/TicketCard.tsx

```diff
@@ -0,0 +1,39 @@
+import "@fortawesome/fontawesome-svg-core/styles.css";
+import { faCheck, faWarehouse } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
```

Comment:
Fine for now Font Awesome, but there is also HeroIcons. Unsure of the commercial Licensing's on FontAwesome. 
https://heroicons.com/